### ISSUE TITLE
broid-alexa not binding to Herokuport

### ISSUE THREAD

**Issue by ghost** [broidHQ/integrations#83]
```
{"pid":28,"hostname":"d283f4b4-47a6-41fb-84b5-dfaf5737e04d","name":"webhook_server","level":30,"time":1489420541677,"msg":"Server listening at port 127.0.0.1:5715...","v":1}
2017-03-13T15:56:37.536074+00:00 heroku[web.1]: Error R10 (Boot timeout) -> Web process failed to bind to $PORT within 60 seconds of launch
2017-03-13T15:56:37.536138+00:00 heroku[web.1]: Stopping process with SIGKILL
2017-03-13T15:56:37.692355+00:00 heroku[web.1]: Process exited with status 137
2017-03-13T15:56:37.726319+00:00 heroku[web.1]: State changed from starting to crashed
2017-03-13T15:56:39.641632+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=opinionated-bot.herokuapp.com request_id=96877784-dfbc-4a58-be28-c4452606b74d fwd="80.169.194.69" dyno= connect= service= status=503 bytes= protocol=https
2017-03-13T15:56:39.770329+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=opinionated-bot.herokuapp.com request_id=6d26ca37-181b-42ed-b5b2-1e915129af29 fwd="80.169.194.69" dyno= connect= service= status=503 bytes= protocol=https
```

When running broidAlexa on heroku I get this error, the port and host are correct but Heroku things it's not listening.

**Comment by dustinblackman** [comment]
`5715` seems low for a Heroku port, I remember them using larger numbers. What does your initialization of Broid Alexa look like?

**Comment by killix** [comment]
should be fixed with the version 2.2.0 as you can now use your http server
